IdentityCore/tests/util/MSIDTestCacheDataSource.h

```diff
@@ -23,7 +23,7 @@
 
 #import <Foundation/Foundation.h>
 #import "MSIDExtendedTokenCacheDataSource.h"
-#import "MSIDBaseToken.h"
+#import "MSIDCredentialType.h"
 #import "MSIDCacheItemSerializing.h"
 
```

Comment:
[The CI is failing](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=1336850&view=logs&j=d46b93ad-1be7-5eab-6980-c78d65372179&t=0153d404-5636-5846-4ebd-361eb75124ec) here: `IdentityCore/src/workplacejoin/MSAIMSIDWorkPlaceJoinUtilBase`.

Does it mean we need to update [WorkplaceJoin](https://github.com/AzureAD/WorkplaceJoin-for-iOS) to 10.15 too?

Reply:
Now it's working, not sure what caused it to fail. Apparently it is updated to 10.15